src/main/java/tasks/Todo.java

```diff
@@ -0,0 +1,21 @@
+package tasks;
+
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+
+    private Todo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    @Override
+    public Todo clone() {
+        return new Todo(super.description, super.isDone);
+    }
```

Comment:
Interesting method, perhaps a header comment would be helpful for me to understand why you included it :)